taskmanager/features/task.feature

```diff
@@ -72,5 +72,5 @@ Feature: Task management
 
   Scenario: Change task status from CLOSED to TODO
     Given there is a task named "Add switch language button" with "CLOSED" status
-    When user named "Sarah" changes task named "Add switch language button" status to "TODO"
-    Then task named "Add switch language button" should have status "CLOSED"
+    When task "UnexpectedStatusChangeException" is thrown during user named "Sarah" is changing task named "Add switch language button" status change to "TODO"
```

Comment:
code implementation looks quite fine but according to our discussion you should not expose implementation details in behat scenarios.